docs/data_preparation.rst

```diff
@@ -455,37 +420,29 @@ Processing
     # 12: cdsEnd (52314677)
     # 14: exonStarts (52301201,52306253,52306882,52307342,52307757,52308222,52309008,52309819,52312768,52314542,)
     # 15: exonEnds (52301479,52306319,52307134,52307554,52307857,52308369,52309284,52310017,52312899,52317145,)
-
     join -t $'\t' \
-        ~/data/genbank-data/${ASSEMBLY}/gene_subinfo_citation_count \
-        ~/data/genbank-data/${ASSEMBLY}/geneid_refGene_count \
+        $DATADIR/$ASSEMBLY/gene_subinfo_citation_count \
+        $DATADIR/$ASSEMBLY/geneid_refGene_count \
         | awk -F $'\t' '{print $7 "\t" $9 "\t" $10 "\t" $2 "\t" $16 "\t" $8 "\t" $6 "\t" $1 "\t" $3 "\t" $4 "\t" $11 "\t" $12 "\t" $14 "\t" $15}' \
-        > ~/data/genbank-data/${ASSEMBLY}/geneAnnotations.bed
-    head ~/data/genbank-data/${ASSEMBLY}/geneAnnotations.bed
-    wc -l ~/data/genbank-data/${ASSEMBLY}/geneAnnotations.bed
+        > $DATADIR/$ASSEMBLY/geneAnnotations.bed
 
-    python scripts/exonU.py \
-        ~/data/genbank-data/${ASSEMBLY}/geneAnnotations.bed \
-        > ~/data/genbank-data/${ASSEMBLY}/geneAnnotationsExonUnions.bed
-    wc -l ~/data/genbank-data/${ASSEMBLY}/geneAnnotationsExonUnions.bed
+    # Download: https://raw.githubusercontent.com/higlass/clodius/develop/scripts/exonU.py
+    python exonU.py $DATADIR/$ASSEMBLY/geneAnnotations.bed > $DATADIR/$ASSEMBLY/geneAnnotationsExonUnions.bed
 
-Creating a HiGlass Track
-^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. code-block:: bash
+Create a gene annotation track file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    workon hg-server
-    ASSEMBLY=mm9
+.. code-block:: bash
 
     clodius aggregate bedfile \
-        --max-per-tile 20 --importance-column 5 \
-        --assembly ${ASSEMBLY} \
-        --output-file ~/data/tiled-data/gene-annotations-${ASSEMBLY}.db \
+        --max-per-tile 20 \
+        --importance-column 5 \
+        --assembly $ASSEMBLY \
+        --output-file $DATADIR/$ASSEMBLY/gene-annotations-${ASSEMBLY}.db \
         --delimiter $'\t' \
-        ~/data/genbank-data/${ASSEMBLY}/geneAnnotationsExonUnions.bed 
+        $DATADIR/$ASSEMBLY/geneAnnotationsExonUnions.bed 
 
-    aws s3 cp ~/data/tiled-data/gene-annotations-${ASSEMBLY}.db \
-        s3://pkerp/public/hg-server/data/${ASSEMBLY}/
 
 Importing into HiGlass
```

Comment:
Why are these authenticated form requests to `higlass.io` here?

Reply:
They shouldn't be. Thanks for deleting them 😬

Reply:
I haven't yet :P. I noticed them after making the PR...

Reply:
Now they're gone :)